packages/cron/src/jobs/pins.js

```diff
@@ -63,9 +64,22 @@ export async function updatePinStatuses ({ cluster, db }) {
         return null // not tracked by our cluster
       }
 
-      const status = toPinStatusEnum(peerMap[pin.location.peerId].status)
+      let status = toPinStatusEnum(peerMap[pin.location.peerId].status)
 
       if (status !== 'Pinned' && status !== 'Remote') {
+        const cidV0 = denormalizeCid(pin.contentCid)
```

Comment:
Not all CIDs can be downgraded. You need to try/catch here (or do the check).

See https://github.com/multiformats/cid#cidv0

As we've discussed, the likelyhood is high that this is a v0 CID but we can't guarantee. We should not halt processing in the exceptional case.